src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static final String boundary = "____________________________________________________________";
+
+    public static void main(String[] args){
+
+        Scanner scan = new Scanner(System.in);
+        Task[] item = new Task[100];
+        int taskno = 0;
+        int index = 1;
+        boolean EndConvo = false;
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("\nHello from\n" + logo);
+
+        System.out.println(boundary+"\nHello! I'm Duke\n" + "What can I do for you?\n" +boundary);
+
+
+        while(!EndConvo){
+            if(!scan.hasNextLine()){
+                return;
+            } else{
+                String CommandTxt = scan.nextLine();
+                String[] Message = CommandTxt.split(" ", 2);
```

Comment:
Again, the naming  should be in camelCase like if just one word, it would be "message" for such